packages/contracts-bedrock/src/L1/RoundRobinLeaderElection.sol

```diff
@@ -31,7 +30,9 @@ contract RoundRobinLeaderElection is LeaderElectionBatchInbox, OwnableUpgradeabl
         creation_block_number = block.number;
     }
 
-    function addParticipant(address _addr) public onlyOwner {
+    // TODO for production purposes it might be desirable to make this function "onlyOwner". However the tests would be
+    // harder to write
```

Comment:
Is it a lot of work to make this work in the tests? I'm not sure we should take this shortcut.

Reply:
Maybe not. I get the error " Ownable: caller is not the owner" if the modifier onlyOwner is set. So we would need to be able to identify / pass the private key used in [initializeRoundRobinLeaderElection](https://github.com/EspressoSystems/op-leader-election/blob/1afacdb45cf39a3a6346dd0b41cc3a1cd0830b62/packages/contracts-bedrock/scripts/Deploy.s.sol#L773).

Reply:
As discussed, this does not look trivial. https://github.com/EspressoSystems/op-leader-election/issues/73